gamemodes/terrortown/gamemode/client/cl_help_data.lua

```diff
@@ -0,0 +1,253 @@
+---
+-- @author Mineotopia
+
+HELP_DATA = {}
+
+local HELP_DATA_OBJECT = {}
+
+---
+-- Binds data table to the @{HELP_DATA} object
+-- @param table data The data table with all submenues
+-- @return @{HELP_DATA} The object to be used in the hook
+-- @internal
+-- @realm client
+function HELP_DATA:BindData(menuTbl)
+	self.menuTbl = menuTbl or {}
+
+	return self
+end
+
+---
+-- Creates a new submenu in the main menu
+-- @param string id The unique ID of the submenu
+-- @return table A reference to the new table
+-- @realm client
+function HELP_DATA:RegisterSubMenu(id)
+	if self:Exists(id) then return end
+
+	local pos = #self.menuTbl + 1
+
+	self.menuTbl[pos] = table.Copy(HELP_DATA_OBJECT)
+	self.menuTbl[pos].id = id
+
+	return self.menuTbl[pos]
+end
+
+---
+-- Checks if a menu with the given id is already registered
+-- @param string id The unique menu identifier
+-- @return boolean Returns if the identifier is already used
+-- @realm client
+function HELP_DATA:Exists(id)
+	for i = 1, #self.menuTbl do
+		if self.menuTbl[i].id == id then
+			return true
+		end
+	end
+
+	return false
+end
+
+---
+-- Returns a list of all registered menues that are not admin only and
+-- the should show function does not return false
+-- @return table A table of all menues
+-- @internal
+-- @realm client
+function HELP_DATA:GetNormalMenues()
+	local menuTbl = {}
+
+	for i = 1, #self.menuTbl do
+		local menu = self.menuTbl[i]
+
+		-- do not show if it is admin only
+		if menu.adminOnly then continue end
+
+		-- do not show if the shouldShow function returns false
+		if isfunction(menu.shouldShowFn) and menu.shouldShowFn() == false then continue end
```

Comment:
```suggestion
		if isfunction(menu.shouldShowFn) and not menu.shouldShowFn() then continue end
```

Reply:
Same here. It should only be hidden if `false` is returned